src/utils.ts

```diff
@@ -139,6 +274,16 @@ export function getDosageInstruction(
   return null;
 }
 
+/**
```

Comment:
The rest are all new utility methods that hopefully are understandable based on the comments.